moksha.hub/moksha/hub/hub.py

```diff
@@ -223,7 +223,10 @@ def consume_stomp_message(self, message):
 
         # FIXME: only do this if the consumer wants it `jsonified`
         try:
-            body = JSON.loads(message['body'])
+            if message['body']:
+                body = JSON.loads(message['body'])
+            else:
+                body = message['body']
```

Comment:
Here, body should probably be `body = {}`, no?  I think subsequent Consumer modules will expect a dict-type object.

Reply:
I was trying to keep things consistent with the current behavior, but if you think an empty dict makes more sense, happy to make that change.